basics/ManualTestProject.txt

```diff
@@ -0,0 +1,303 @@
+7. if chosen product shows up on check out page summary then whole test passed
+
+
+
+Test case ID: 019
+Related functional requirements: FR007 - B4
+Title: buyer add items in shopping cart
+Summary: User will be able to add items in shopping cart.
+Precondition: user has an account on this website and logged in and has more than one items in shopping cart
+Steps: 
+No steps avaliable! this website doesn't support adding items in shopping cart
+
+Test case ID: 020
+Related functional requirements: FR007 - B4
+Title: buyer remove items in shopping cart
+Summary: User will be able to remove items in shopping cart.
+Precondition: user has an account on this website and logged in and has more than one items in shopping cart
+Steps: 
+1. click on shopping cart "Cart" on the top right that will jump to summary page
+2. under the "Qty" section, click on "-" button
+3. the amount of this product - 1, test passed. 
+
+Test case ID: 021
+Related functional requirements: FR007 - B4
+Title: buyer update quantity of items in shopping cart
+Summary: User will be able to update quantity of items in shopping cart.
+Precondition: user has an account on this website and logged in and has more than one items in shopping cart
+Steps: 
+1. click on shopping cart "Cart" on the top right that will jump to summary page
+2. under the "Qty" section, click on "+" button
+3. the amount of this product + 1, test passed. 
+
+Test case ID: 022
+Related functional requirements: FR008 - B3
+Title: Buyer enter billing address and shopping address before checkout
+Summary: Buyer will required to enter billing and shipping address before checkout and payment.
+Precondition: user has an account on this website and logged in and has more than one items in shopping cart
+Steps: 
+1. click on shopping cart "Cart" on the top right that will jump to summary page
+2. click on "Proceed to checkout" on the right bottom.
+3. if user has already input their billing address and/or shopping address then by default those addresses will be used. if not then website will ask user to input their addresses
+
+Test case ID: 023
+Related functional requirements: NFR001
+Title: The website repository shall accommodate up to 100 users concurrently.
+Summary: use selenium to write test to ask http request for 100 users concurrently 
+Steps: use selenium to write test to ask http request for 100 users concurrently. use synchronization to simulate 100 user access to home page.
+
+Test case ID: 024
+Related functional requirements: NFR002
+Title: page loading speed test
+Summary: web pages should not take more than 30 seconds to load in good speed of internet.
+Steps: use selienium/java to write a test. this test will simulate 10000 users that try to access to home page. each user will record the time that 
+access to home page. after that will average out the time. if the average time is <= to 30 seconds, then test pass
+
+
+Find and log 2 bugs.
+
+Bug ID: 001
+Title: Searching filter can not update
```

Comment:
Need Severity and Priority fields.
Assigned to.
Steps to reproduce.
Bug State.
Visual Proof (screenshot, video, etc).